cli/src/commands/extensions/api.rs

```diff
@@ -137,7 +138,7 @@ struct ProcessOutput {
 #[op]
 async fn analyze(
     op_state: Rc<RefCell<OpState>>,
-    packages: Vec<PackageDescriptor>,
+    packages: Vec<PackageDescriptorAndLockfilePath>,
```

Comment:
This change needs:

1. An entry in `extensions/CHANGELOG.md`
2. To be documented in the docs for `PhylumApi.analyze()` in `extensions/phylum.ts`